src/plain_data.rs

```diff
@@ -18,7 +18,7 @@
 use rustc_serialize::{Decoder, Encodable, Encoder};
 use xor_name::XorName;
 
-/// PlainData
+/// Plain data with a name and a value.
```

Comment:
Name is set and not derived in this case. Allowing for contents to be set by user i.e. may contain and be restricted to a DSL